content/posts/introducing-slight.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Introducing SpiderLightning - A Cloud System Interface with WebAssembly"
+description: "Announcing SpiderLightning and slight, a distributed application runtime for Wasm featuring portable application building blocks."
+date: 2022-10-31 00:00:00 +0000 UTC
+authorname: "Jiaxiao (Joe) Zhou"
+author: "@jiaxiao_zhou"
+authorlink: "https://twitter.com/jiaxiao_zhou"
+image: "images/logos/twitter-card.png"
+tags: ["slight", "spiderlightning", "Wasm", "WASI"]
+---
+
+At DeisLabs, we are researching and developing various WebAssembly tools and systems overtime. We have been working on a new project called SpiderLightning, which is a distributed application interface written in an Interface Definition Language (IDL) called [WIT](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md). SpiderLightning is paired with a CLI called slight, which we recently released "0.2.0" version. In this blog post, we will introduce SpiderLightning and slight.
```

Comment:
`....which is a set of interfaces and implementations intended to provide developers in the WebAssembly ecosystem a way to use the WebAssembly component model quickly and easily for distributed applications that need only a core set of service functionality. The interfaces are written in an Interface Definition Language (IDL) called [WIT](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md), part of the WebAssembly component model work in the [Bytecode Alliance Foundation (BCA)](https://bytecodealliance.org). The SpiderLightning project also comes with a CLI called slight, and we recently released "0.2.0" version. In this blog post, we will introduce SpiderLightning and slight.`